packages/frontend/src/components/common/TagInput/TagInput.tsx

```diff
@@ -447,6 +458,7 @@ export const TagInput = forwardRef<HTMLInputElement, TagInputProps>(
                 disabled={disabled}
                 // @ts-ignore
                 autoComplete="off"
+                data-1p-ignore
```

Comment:
This is unrelated. Without it one password shows up in this input. It seems like a one-off solution to a problem that might come up with other plugins. But it also seems harmless. 

If we'd rather not do this, I'm happy to remove it.

Reply:
oh, did not know about this one!